From 99639efd12c017445dd2afe5ee10565791b1a1a8 Mon Sep 17 00:00:00 2001
From: Zuohai Xu <zouhaix@marvell.com>
Date: Mon, 20 May 2019 18:24:22 +0800
Subject: [PATCH 239/386] arm64: dts: 9130-crb: Add C45 compatible for 88E2110
 PHY

PHY 88E2110 support C45 clause register access method.

Change-Id: If1cdf89fbb796923dff5f251573a94219ae95f0f
Signed-off-by: Zuohai Xu <zouhaix@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/9261
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/cn9130-crb.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/marvell/cn9130-crb.dtsi b/arch/arm64/boot/dts/marvell/cn9130-crb.dtsi
index 43dd8bb1dc9c..ff52c157ccb7 100644
--- a/arch/arm64/boot/dts/marvell/cn9130-crb.dtsi
+++ b/arch/arm64/boot/dts/marvell/cn9130-crb.dtsi
@@ -142,6 +142,7 @@
 &cp0_xmdio {
 	status = "okay";
 	nbaset_phy0: ethernet-phy@0 {
+		compatible = "ethernet-phy-ieee802.3-c45";
 		reg = <0>;
 	};
 };
-- 
2.17.1

